thirdparty/DiskANN/src/index.cpp

```diff
@@ -1714,11 +1713,11 @@ namespace diskann {
                              Parameters &             parameters,
                              const std::vector<TagT> &tags) {
     if (!file_exists(filename)) {
-      diskann::cerr << "Data file " << filename
-                    << " does not exist!!! Exiting...." << std::endl;
+      // diskann::cerr << "Data file " << filename
+                    // << " does not exist!!! Exiting...." << std::endl;
       std::stringstream stream;
       stream << "Data file " << filename << " does not exist." << std::endl;
-      diskann::cerr << stream.str() << std::endl;
```

Comment:
Same, do we need stream here?

Reply:
It's from the original code of diskann. Just change the output approach from `cout` to `LOG`